dz/css/menuStyle.css

```
@@ -0,0 +1,166 @@
+html, body {
+    height: 100%;
+    margin: 2px;
+    overflow:  hidden;
+}
+
+.container {
+    display: -webkit-flex;
+    display: flex;
+    height: 100%;
+}
+
+.nav-default {
+    background: var(--nav-bg-color);
+    height: 100%;
+    width: 350px;
+    max-width: 350px;
+    margin-right: 250px;
+}
+
+.title {
+    background-color: var(--main-btn-color);
+    padding-left: 10px;
+    height: 45px;
+    margin-bottom: 120px;
+}
+
+.title img {   
+    height: 100%;
+    width: 30%;
+    min-width: 60px;
+}
+
+.nav-items {
+    padding: 5px;
+}
+
+.btn {
+    outline: none; 
+}
+
+.nav-items .btn {
+    width: 100%;
+    background-color: var(--nav-bg-color);
+    border: 1px solid var(--border-color);
+    border-radius: 4px;   
+    height: 50px;
+    min-height: 50px;
+    color: black;    
+    font-weight: bold;
+    margin-bottom: 30px;
+    display: flex;
+}
+
+.btn img { 
+    margin-right: 10px;
+}
+
+.nav-items .active {
+    background-color: var(--main-btn-color);
+    color: var(--bg-color);
+}
+
+.about {
+    text-align: center;
+    color: blue;
+}
+
+.main-block {
+    width: 500px;
+    justify-content: center;
+}
+
+.main-block h2 {
+    font-weight: bold;
+    width: 80%;
+    font-size: 32px;
+    margin-bottom: 20px;
+}
+
+.input-default {
+    width: 80%;
+    padding: 12px 20px;
+    margin: 10px 0;
+    box-sizing: border-box;
+    border: 1px solid var(--border-color);
+    border-radius: 4px;
+    outline: none; 
+}
+
+.submit {
+    width: 50%;
+    background-color: var(--main-btn-color);
+    border: none;
+    border-radius: 4px;   
+    height: 45px;
+    color: var(--bg-color);    
+    margin-bottom: 10px;
+    font-weight: bold;
+    font-size: 17px;
+    display: flex;
+    min-width: 210px;
+}
+
+.input-area { 
+    margin-bottom: 8px;
+}
+
+.full-width {
+    width: 100%;
+}
+
+.password-block {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    position: relative;
+    width: 80%;
+}
+
+.password-block img {
+    position: absolute;
+    z-index: 100;
+    right: 15px;
+    padding: 0px 0px 0px 10px;

```

Если значение какого-то CSS-свойства равно 0, единицы измерения не нужно указывать:
```suggestion
    padding: 0 0 0 10px;
```